docs/_guide/styles.md

```diff
@@ -113,19 +113,19 @@ To define a static `styles` property:
 
 Static styles apply to all instances of an element. Any expressions in your CSS are evaluated and included **once**, then reused for all instances. 
 
-For security reasons, expressions must be tagged with the `cssLiteral` template literal tag:
+For security reasons, use **css** to pass non-literal values using only trusted sources:
```

Comment:
This appears to be the description for `unsafeCss`. You can't use `css` with non-literal values, and "trusted sources" doesn't make sense for a literal. Correct me if I'm wrong here, @justinfagnani

Reply:
I refactored this a bit to hopefully provide coverage and clarity, but I may be missing something that is planned that will change this, such as #626. Sorry about dragging out what I thought would be a simple doc change. Please advise if I need to address something else here.